### PR TITLE
Revision 1.5.2

### PR DESCRIPTION
**Pull request by Dreyal**
Changelog from https://steamcommunity.com/sharedfiles/filedetails/changelog/2077825026

Cloaked/camouflaged units now become more visible, and show up on minimap at proximity (L1/L2/L3 distances of 12/ 8/ 4).
View model now reflects camouflage level (less visible at higher levels).
Shade now gives level 3 cloaking (was 2).
Cloak-in rate changed to 3 and no longer depends on cloak level.
Ink buff duration increased from 0.6 to 0.99 second.
Minor tweaks to shader and cloaking variables.
Disabled Hallucinated Hive and Harvester spawning.

Cloaked player embryos are now fully invisible like Eggs.
Changed minimap cloaked visible distance from 12/ 8/ 4 to 8/ 6 /4

Increased visibility of Detected cloaked units

Drifter Egg now fully cloaks
Increased cloak delay in Ink from 0.5 to 0.8s

### CODE REVIEW

File: src/lua/CommunityBalanceMod/Classes/Marine/Marine.lua
Comment (adsfgg) [inline src/lua/CommunityBalanceMod/Classes/Marine/Marine.lua:1]: Can we remove Marine.lua from FileHooks.lua as well?

File: docs/changelog.md
Comment (adsfgg) [inline docs/changelog.md:1]: Could we update the main changelog with these changes as well?

**Review by adsfgg — CHANGES_REQUESTED** [review]

File: src/lua/CommunityBalanceMod/FileHooks.lua [hunk 1]
```diff
@@ -3,6 +3,11 @@ local defaultConfig = {
     build_tag = "dev"
 }
 
+local kVersion = "1.5.2"
```
Comment (adsfgg) [inline src/lua/CommunityBalanceMod/FileHooks.lua:6]: Can we get this from g_communityBalanceModConfig instead of hardcoding it?

**Review by adsfgg — CHANGES_REQUESTED** [review]

File: src/lua/CommunityBalanceMod/FileHooks.lua (hunk 1 above)
Comment (Dreyal) [reply src/lua/CommunityBalanceMod/FileHooks.lua]: Done.

**Review by adsfgg — APPROVED** [review]

**Pull request merged**